src/components/Game/Submit/Submit.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 
 import classes from './Submit.scss';
 
-const Submit = ({ onSubmit, onCancel, dragDisabled }) => (
+const Submit = ({ onSubmit, onCancel, isCancelBtnVisible }) => (
```

Comment:
`showCancel`?
visibility is often associated with `visibility` style property (https://developer.mozilla.org/en-US/docs/Web/CSS/visibility) and that is not the accurate term in this case